Plugins/FormatSwift/Plugin.swift

```diff
@@ -166,3 +169,92 @@ enum CommandError: Error {
   case lintFailure
   case unknownError(exitCode: Int32)
 }
+
+// MARK: - SwiftVersion
+
+struct SwiftVersion: Comparable {
```

Comment:
The changes in this PR depend on https://github.com/airbnb/swift/pull/235 for CI to pass

Reply:
I'll rebase and remove these changes from this PR once https://github.com/airbnb/swift/pull/235 is merged.